shell/platform/darwin/macos/framework/Source/FlutterMetalCompositor.mm

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterMetalCompositor.h"
+
+namespace flutter {
+
+FlutterMetalCompositor::FlutterMetalCompositor(FlutterViewController* view_controller)
+    : FlutterCompositor(view_controller) {}
+
+bool FlutterMetalCompositor::CreateBackingStore(const FlutterBackingStoreConfig* config,
+                                                FlutterBackingStore* backing_store_out) {
+  return false;
```

Comment:
can you add a `FML_CHECK(false) << "Not implemented, see issue: "` and file an issue, it might be confusing for users to see this in the interim.